cli/tools/test/fmt.rs

```diff
@@ -88,10 +89,19 @@ pub fn format_test_error(
     .exception_message
     .trim_start_matches("Uncaught ")
     .to_string();
-  if options.hide_stacktraces {
-    return js_error.exception_message;
+  let message = if options.hide_stacktraces {
+    js_error.exception_message
+  } else {
+    format_js_error(&js_error)
+  };
+  // This is a first pass route. I think that the exception message should always
+  // be free of ANSI codes anyway so it would be more efficient to alter `format_js_error`
+  // to not print ANSI codes if `options.strip_ascii_color` is set to `true`.
```

Comment:
Can you please remove this comment - I don't understand the "first pass route" bit so I don't think this comment is useful